### ISSUE TITLE
class with virtual methods does not work

### ISSUE THREAD

**Issue by gastald88** [awulkiew/graphical-debugging#58]
I have a class representing a point which also have a couple of virtual methods. 

I set up a user defined xml element to watch it but I cannot visualize objects of such type. Once I remove the virtual methods, objects of this class are correctly showed in the geometry watch window.
Are classes with virtual methods not supported?

Thanks for the very useful extension anyway!

**Comment by awulkiew** [comment]
This works for me:
```
struct PolyGeometry
{
    virtual void foo() {}
};

struct PolyPt : PolyGeometry
{
    PolyPt(double x, double y) : x(x), y(y) {}
    void foo() override {}
    virtual void bar() {}
    double x, y;
};

PolyPt poly_pt(1, 2);
```

```
	<Point Type="PolyPt" CoordinateSystem="Cartesian" Unit="None">
		<Coordinates>
			<X>x</X>
			<Y>y</Y>
		</Coordinates>
	</Point>
```
![obraz](https://github.com/awulkiew/graphical-debugging/assets/1226951/a92f87d3-6fd9-4154-a99f-6d37b612e8d8)

**Comment by gastald88** [comment]
I see, I may add that the class with virtual methods comes from a dll library that I am linking, that's the only difference I can see so far. I will try harder to make this issue of mine replicable.

**Comment by awulkiew** [comment]
What is the type displayed in the type column exactly (the whole thing)?

**Comment by gastald88** [comment]
That's exactly the point. And once I remove the virtual methods, I see only `MyType` in the type column and everything works.

So for the time being it is not supported, hope it will be in the future, it would be very useful.

Thanks again for the support!

**Comment by awulkiew** [comment]
I added the support. It will be released with v0.54.

**Comment by gastald88** [comment]
Very good job, thanks!
